controlSystem/RecoveryBoard/firmware-rs/Cargo.toml

```diff
@@ -21,6 +21,17 @@ embedded-hal-nb = "1.0.0"
 noline = "0.5.1"
 heapless = "0.9.1"
 format_no_std = "1.2.0"
+embedded-test-linker-script = { version = "0.1.0" }
```

Comment:
Non-blocking, but this will help me come up to speed.  I want to comment on Cargo.toml line 2:

 name = "firmware-rs"

to ask whether this name needs to match the directory in which this toml file lives?  Unfortunately Github doesn't allow me to comment on lines it deems have not changed across the, would it be most recent two commits?